runhouse/servers/cluster_servlet.py

```diff
@@ -2,6 +2,7 @@
 import copy
 import datetime
 import json
+import os.path
```

Comment:
dumb nit - can we use `Path(file_path).exists()` instead - just for consistency?